scheduler-driver/src/main/java/gov/nasa/jpl/aerie/scheduler/constraints/activities/ActivityCreationTemplate.java

```diff
@@ -160,8 +160,9 @@ protected ActivityCreationTemplate fill(final ActivityCreationTemplate template)
                                      + " because its DurationType is Uncontrollable");
         }
         if(this.acceptableAbsoluteTimingError.isZero()){
-          logger.warn("Root-finding is likely to fail as activity has an uncontrollable duration and the timing "
-          + "precision is 0. Setting it for you at 1s. Next time, use withTimingPrecision() when building the template.");
+          //TODO: uncomment when precision can be set by user
+          //logger.warn("Root-finding is likely to fail as activity has an uncontrollable duration and the timing "
+          //+ "precision is 0. Setting it for you at 1s. Next time, use withTimingPrecision() when building the template.");
           this.acceptableAbsoluteTimingError = Duration.of(500, Duration.MILLISECOND);
```

Comment:
Quick question based on the logger text that was commented out: Is the acceptableAbsoluteTimingError meant to be half of the timing precision? Because the text says it's setting the precision to 1s, but `acceptableAbsoluteTimingError` is set to .5s.

Reply:
We want to represent the acceptable error around the desired value x, such that the result falls into [x-error, x+error]. The total error will be 0.5 * 2. We can change the sentence if you come up with something better.

Reply:
This can wait until we're adding back the logger warning, but based on your explanation it sounds like the precision is actually 0.5s (as the acceptable range is +-0.5s)